src/main/java/org/wso2/carbon/connector/azure/storage/BlobUploader.java

```diff
@@ -64,6 +64,10 @@ public void connect(MessageContext messageContext) {
             CloudBlockBlob blob = container.getBlockBlobReference(fileName);
             File sourceFile = new File(filePath);
             fileInputStream = new FileInputStream(sourceFile);
+            if (messageContext.getProperty(AzureConstants.BLOBCONTENTTYPE)!=null){
```

Comment:
Shall we format this ? 
```suggestion
            if (messageContext.getProperty(AzureConstants.BLOBCONTENTTYPE) != null) {
```